server/db/index.js

```diff
@@ -5,7 +5,7 @@
 const mongoose = require('mongoose');
 mongoose.Promise = global.Promise;
 let MONGO_URL;
-const MONGO_LOCAL_URL = 'mongodb://localhost/wishlist'
+const MONGO_LOCAL_URL = 'mongodb+srv://mistmr7:axkeFkMza7Yt1iq9@cluster0-3ioar.azure.mongodb.net/Wish?retryWrites=true';
```

Comment:
This might be specific to your mongo db connection - I would suggest changing back to the localhost/ wishlist or it does not matter ... we will change in prod env...